tensorflow/lite/micro/benchmarks/Makefile.inc

```diff
@@ -9,21 +9,23 @@ tensorflow/lite/micro/benchmarks/micro_benchmark.h
 PERSON_DETECTION_BENCHMARK_SRCS := \
 tensorflow/lite/micro/benchmarks/person_detection_benchmark.cc
 
+PERSON_DETECTION_BENCHMARK_GENERATOR_INPUTS := \
+tensorflow/lite/micro/examples/person_detection/testdata/person.bmp \
+tensorflow/lite/micro/examples/person_detection/testdata/no_person.bmp
+
 ifeq ($(CO_PROCESSOR),ethos_u)
+  PERSON_DETECTION_BENCHMARK_GENERATOR_INPUTS += \
+    tensorflow/lite/micro/models/person_detect.tflite
+else
   # Ethos-U use a Vela optimized version of the original model.
   PERSON_DETECTION_BENCHMARK_SRCS += \
-  $(MAKEFILE_DIR)/downloads/person_model_int8/person_detect_model_data_vela.cc
+  $(GENERATED_SRCS_DIR)tensorflow/lite/micro/models/person_detect_model_data_vela.cc
```

Comment:
nit and totally up to you:  $(GENERATED_SRCS_DIR)/tensorflow would more clearly indicate the expectation.  I think // would do no harm. WDYT?

Reply:
The // resulted in the vela compiler's outputs paths being a bit odd. More importantly, we've had double slash cause the MacOS build to fail previously so I am trying to avoid it.

I do agree with you that having `GENERATED_SRCS_DIR` not have a trailing / would be nicer. But that is how the Makefiles have been written so we'd have to make the change in multiple places.

Reply:
For completeness, here is the MacOS fix: https://github.com/tensorflow/tflite-micro/pull/367